code/datums/emergency_calls/cmb.dm

```diff
@@ -0,0 +1,200 @@
+		to_chat(M, SPAN_BOLD("You are [pick(40; "skeptical", 30;"ambicable", 30;"supportive")] of the USCM's actions on the frontier."))
+		to_chat(M, SPAN_BOLD("Through a lot of hard work, your organization managed to convince the Colonial Marshals to take you to the frontier for an article about the quality of life there."))
+		to_chat(M, SPAN_BOLD("Observe the Feds in their adventures and keep an eye out for any inhumane acts from others. The Neroid Sector is full of atrocities on every side."))
+		to_chat(M, SPAN_BOLD("Do not instigate or start any confrontations. You are an observer, and you do not wage wars. Only intervene in medical emergencies."))
+		to_chat(M, SPAN_BOLD("You were on your way to a crime scene when your ship was diverted to the [MAIN_SHIP_NAME] because of a distress beacon."))
+	else
+		to_chat(M, SPAN_BOLD("You are a CMB Deputy, originally from [pick(70;"The United Americas", 20;"Sol", 10;"a colony on the frontier")]."))
+		to_chat(M, SPAN_BOLD("You joined the Marshals through [pick(50; "pursuing a career during college", 40;"working for law enforcement", 10;"being recruited for your skills")]."))
+		to_chat(M, SPAN_BOLD("Following the lead of your Marshal, you have become renown for your steadfast commitment to justice, fighting against crime and corruption alike."))
+		to_chat(M, SPAN_BOLD("While enroute to an investigation you were diverted by your command at Anchorpoint Station to the [MAIN_SHIP_NAME] because of a distress beacon."))
+		to_chat(M, SPAN_BOLD("You have been stationed at Anchorpoint Station for [pick(80;"several months", 10;"only a week", 10;"years")] investigating henious crimes among the frontier."))
+		to_chat(M, SPAN_BOLD("The laws of arth stretch beyond the Sol. Where others fall to corruption, you stay steadfast in your morals."))
+		to_chat(M, SPAN_BOLD("Corporate Officers chase after paychecks and promotions, but you are motivated to do your sworn duty and care for the population, no matter how far or isolated a colony may be."))
+		to_chat(M, SPAN_BOLD("Despite being stretched thin, the stalwart oath of the Marshals has continued to keep communities safe, with the CMB well respected by many. You are a representation of that oath, serve with distinction."))
+
+// Anchorpoint Station Colonial Marines, use this primarily for reinforcing or evacuating the CMB, as the CMB themselves are not equipped to handle heavy engagements.
+/datum/emergency_call/CMB/anchorpoint
+	name = "CMB - Anchorpoint Station Colonial Marine QRF (Friendly)"
+	mob_max = 6
+	mob_min = 3
+	max_medics = 1
+	max_engineers = 1
+	max_smartgunners = 1
+	probability = 0
+
+/datum/emergency_call/CMB/anchorpoint/New()
+	..()
+	arrival_message = "Incoming Transmission: [MAIN_SHIP_NAME], This is Anchorpoint Station. Be advised, a QRF Team our Colonial Marines is attempting to board you. Open your ports, transmitting docking codes now. Standby."
```

Comment:
```suggestion
	arrival_message = "Incoming Transmission: [MAIN_SHIP_NAME], this is Anchorpoint Station. Be advised, a QRF Team of Colonial Marines is attempting to board you. Open your ports, transmitting docking codes now. Standby."
```

also this doesn't need to be in new and can just be done in /datum/emergency_call/CMB/anchorpoint